lametro/events.py

```diff
@@ -1,32 +1,47 @@
 from legistar.events import LegistarAPIEventScraper
+from legistar.events import LegistarEventsScraper
 
 import requests
+import lxml.html
+from datetime import datetime
 
 from pupa.scrape import Scraper
 from pupa.scrape import Event
 
 class LametroEventScraper(LegistarAPIEventScraper):
     BASE_URL = 'http://webapi.legistar.com/v1/metro'
     WEB_URL = 'https://metro.legistar.com'
+    EVENTSPAGE = "https://metro.legistar.com/Calendar.aspx"
     TIMEZONE = "America/Los_Angeles"
 
     def scrape(self):
+        web_results = self.scrapeWebCalendar()
+
         for event in self.events():
+            # Create a key for lookups in the web_results dict.
+            key = (event['EventBodyName'].strip(), datetime.strptime(event['EventDate'][:10], '%Y-%m-%d').strftime('%-m/%-d/%Y'), event['EventTime'])
```

Comment:
I changed this: the output of scrapeWebCalendar returns a datetime.datetime class and so does the scrape function. I'll push those changes.